README.md

```diff
@@ -41,6 +41,16 @@ To run tests, make sure that you are in the project root folder and do:
 1. `pip install -r dev-requirements.txt`
 2. `pytest tests/`
 
+# Development environment
+
+To run the widget and APIs in a development environment:
+1. Open a new terminal, then from within the root folder do `npm start`. This will host the widget with `webpack-dev-server`.
+2. Open a new terminal, then from within the root folder do `flask run`. This will start the Flask server for the APIs used by the widget.
```

Comment:
Can you also specify options for how one should run this for when you want to develop against an ip other than localhost?